### PR TITLE
[Bot] Update docs for latest fastlane release (actions.md, available-plugins.md) 🚀

### PR DESCRIPTION
**Pull request by taquitos**
Auto-generated by _fastlane_ bot 🤖

### CODE REVIEW

File: docs/actions.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<!--
+This file is auto-generated and will be re-generated every time the docs are updated.
+To modify it, go to its source at https://github.com/fastlane/fastlane.
+-->
+
```
Comment (revolter) [inline docs/actions.md]: It looks like my PR works <3

**Comment by revolter** [comment]
The comment is missing from `docs/plugins/available-plugins.md`. Do you know which part of the code generates it, so I can open a new PR?

**Comment by taquitos** [comment]
@revolter it's from https://github.com/fastlane/fastlane/blob/master/fastlane/actions/plugin_scores.rb
but specifically, that file is passed in as a param: `params[:output_path]`

**Comment by revolter** [comment]
@taquitos, I think it's because https://github.com/fastlane/fastlane/pull/10910, so I opened #462.

**Comment by janpio** [comment]
(Triggered a retest on circleci with current result from #462 - let's see who is right in there 🗡 )

**Comment by revolter** [comment]
@janpio,

1. How?
2. It would most probably work, because the case in which I personally think it doesn't (returns ok when it shouldn't) is:

```
test

<!-- comment -->
```

**Comment by revolter** [comment]
Shouldn't this be rebased on top of `master`?

**Comment by janpio** [comment]
Oh right... sorry. I ran the same tests again then ;)
Do that, then it will automatically test again.

**Comment by taquitos** [comment]
Sent a new PR and merged that already

**Pull request closed without merging**